docs/Networking.asciidoc

```diff
@@ -5,16 +5,20 @@
 :toclevels: 6
 :author: openQA Team
 
-IMPORTANT: This overview is valid only when using the QEMU backend!
-
-The networking type used is controlled by the `NICTYPE` variable. If unset or
-empty `NICTYPE` defaults to `user`, i.e. QEMU user networking which requires
-no further configuration.
+For tests using the QEMU backend the networking type used is controlled by the
+`NICTYPE` variable. If unset or empty `NICTYPE` defaults to `user`, i.e.
+<<QEMU User Networking>> which requires no further configuration.
 
 For more advanced setups or tests that require multiple jobs to be in the same
 networking the <<TAP based network,TAP>> or <<VDE Based Network,VDE>> based
 modes can be used.
 
+Other backends can be treated just the same as bare-metal setups. Tests can be
+triggered in parallel same for QEMU based ones and synchronization primitives
```

Comment:
Is some connecting word between "parallel" and "same" missing?

Reply:
Actually the word "as" is missing after "same" :)